build.sh

```diff
@@ -21,9 +36,9 @@ time xcodebuild clean test \
     | tee build.log \
     | xcpretty &&
 cat build.log | sh debug-time-function-bodies.sh &&
-rm -rf $DERIVED_DATA &&
 
 # Run test on tvOS
+rm -rf $DERIVED_DATA &&
```

Comment:
Please add a comment explaining why the order Derived Data clearing is important.